html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-fg.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<title>document wg color</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-fgcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html">
```

Comment:
@marcoscaceres i think we do not need to add one more link for help. What do you think about it?

Reply:
Actually, we probably want to link all of the values:

https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-linkcolor
https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-vlinkcolor
https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-alinkcolor

And so on... as this test file is for each of them.